crates/api_models/src/enums.rs

```diff
@@ -264,6 +265,8 @@ pub enum Currency {
 #[strum(serialize_all = "snake_case")]
 pub enum EventType {
     PaymentSucceeded,
+    PaymentProcessing,
```

Comment:
When payment is created , we will mark the status as `authentication_pending` and once payment is made, we will get a `payment_processing` webhook from their side which notifies that the payment is done by customer by not verifed by the network, so now we have change the `authentication_pending` state to `processing` state, otherwise it will be in `authentication_pending` state and customer will make another payment again.